lib/streamlit/delta_generator.py

```diff
@@ -422,10 +465,6 @@ def beta_expander(self, label=None, expanded=False):
             The default state for the expander.
             Defaults to False
 
-        Returns
```

Comment:
I figured since st.empty() is easy enough to understand without a redundant "Returns" line, we can skip it for st.container() and st.expander() too.

(incidentally we get to dodge explaining exactly what a DeltaGenerator is)